src/components/app-container.tsx

```diff
@@ -85,14 +85,19 @@ export class AppContainerComponent extends BaseComponent<IProps, {}> {
 
   private renderComparisonPlaceholder() {
     return (
-      <div className="comparison-placeholder">
+      <div
+        className="comparison-placeholder"
+        onDragOver={(this.handleDragOver)}
+        onDrop={this.handleDrop("comparison")}
+        onMouseOver={this.handleMouseOver}
+      >
         Click or drag an item in the right tabs to show it here
       </div>
     );
   }
 
   private handleDragOver = (e: React.DragEvent<HTMLDivElement>) => {
-    if (e.dataTransfer.getData("workspace.document.key")) {
+    if (e.dataTransfer.types.find((type) => type === "workspace.document.key")) {
```

Comment:
I like the use of `types` to test for availability. I'm going to use that in `DocumentContent`. Note that [Array.prototype.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) isn't available in all browsers (notably IE). It would be straightforward to use [Array.prototype.indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) instead.

Reply:
Ah thanks and good point about `find` being overkill here. We do use `find` elsewhere though - does babel convert that to something IE can run?

Reply:
It wouldn't be converted -- we'd have to have a polyfill in place, which might be true.